cmd/uninstaller.go

```diff
@@ -0,0 +1,28 @@
+package cmd
+
+import (
+	"fmt"
+	"pb/pkg/common"
+	"pb/pkg/installer"
+
+	"github.com/spf13/cobra"
+)
+
+var UnInstallOssCmd = &cobra.Command{
+	Use:     "oss",
+	Short:   "Uninstall Parseable OSS",
```

Comment:
We should have a name of each installation so it is possible to install multiple instances and then delete the one you want to delete.